Taskfile.yml

```diff
@@ -95,3 +96,37 @@ tasks:
     internal: true
     cmds:
       - go test -count 1 -v -run=TestXTykGateway_Lint ./apidef/oas/
+
+  setup:
+    desc: "Setup the project including dependencies and git hooks using lefthooks"
+    cmds:
+      - task: deps:default
+      - lefthook install
+
+  simple-build:
```

Comment:
you could move all or of these new targets from here on under .taskfiles/lint.yml right?